plugin.yml

```diff
@@ -1,11 +1,13 @@
 name: Slapper
 author: jojoe77777
-version: 1.6.1
+version: 1.7.1
 description: Slapper, the NPC plugin for PocketMine-MP
 main: slapper\Main
 api: 3.13.0
```

Comment:
You need to bump the minimum required API version to 3.14.0 since `ItemStackWrapper::legacy` was added there: https://github.com/pmmp/PocketMine-MP/compare/3.13.1...3.14.0#diff-68d4439663f67995bb26425e333d43bbce3d952bbc1df6f26dcdc1173bad458cR41